.github/workflows/compliance.yml

```diff
@@ -59,6 +60,11 @@ jobs:
 
     - name: check-warns
       run: |
+        git log --oneline --first-parent origin/${BASE_REF}..HEAD | grep -E "\[nrf (mergeup|fromlist|toup|noup|temphack|fromtree)\]" || { echo 'Sauce tag missing'; exit 1; }
```

Comment:
The `--first-parent` ensures that, in the case of an upmerge, only the merge commit and any other reverts or out-of-tree commits require the sauce tag